git_commands/git-commands.adoc

```diff
@@ -36,7 +36,7 @@ The `git pull` command updates the local repository from the remote repository,
 
 The remote repository is the Git repository on the network that is the single source of truth for all content being managed under a particular project. Developers use the `git push` command to upload content from the local repository to a remote repository.
 
-[AO: Let's turn the following figure into a real figure.] The following figure shows the working directory, staging environment, local repository, and remote repository locations along with the `git` commands that move data to and from each location. After content has been pulled from a remote repository to a local repository, that content is available in the local filesystem.
```

Comment:
Has this already been done? (I'm not sure if the image below is final or not) If not, we'll have to make sure the request for this image is sent to design ASAP.

Reply:
> Has this already been done? (I'm not sure if the image below is final or not) If not, we'll have to make sure the request for this image is sent to design ASAP.

yes, I guess Athen did the review. I'll look at it first thing in the morning @daleblanc